.github/settings.yml

```diff
@@ -18,6 +18,115 @@ repository:
   allow_rebase_merge: true
 
 labels:
+  # sane labeling scheme
+  - name: 'Priority: Critical'
+    color: "#e11d21"
+    board: false
+    priority: 10
+  - name: 'Priority: High'
+    color: "#eb6420"
+    board: false
+    priority: 9
+  - name: 'Priority: Medium'
+    color: "#fbca04"
+    board: false
+    priority: 8
+  - name: 'Priority: Low'
+    color: "#009800"
+    board: false
+    priority: 7
+  - name: 'Status: Abandoned'
```

Comment:
How is this different to having the ticket closed, maybe with a comment that it was abandoned?

Reply:
A label makes this obvious when listing issues.